about.html

```diff
@@ -0,0 +1,203 @@
+          <div class="hero-header-title">
+            <h3>"Hello! Sharing World"</h3>
+          </div>
+          <div class="hero-title">
+            <h1>International Conference on Sustainable Water Management</h1>
+          </div>
+          <div class="hero-desc">
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaerat
+              culpa illum et neque veritatis, non itaque quasi deleniti nostrum
+              nesciunt. Lorem ipsum dolor sit amet consectetur, adipisicing
+              elit. Velit vel, iusto nostrum explicabo veritatis sint nisi vitae
+              placeat officiis doloremque, soluta officia ut perspiciatis
+              quibusdam quos nam exercitationem optio adipisci.
+            </p>
+          </div>
+          <div class="hero-text-date">
+            <h3>24th - 25th Nov 2022</h3>
+          </div>
+          <div class="hero-bottom">
+            <p>
+              International Conference on Sustainable Water Management (ICSWM)
+            </p>
+            <p class="email">international.conference@conference</p>
+          </div>
+        </div>
+      </section>
+    </header>
+    <!-- MAIN SECTION -->
+    <main class="main-container">
+      <div class="main-top">
+        <div class="conference-title-desc">
+          <h2>
+            International Conference on Sustainable Water Management ( ICSWM )
+          </h2>
+          <p>
+            Lorem ipsum, dolor sit amet consectetur adipisicing elit. Illum,
+            exercitationem maiores quisquam ipsam provident maxime inventore
+            officiis in facere
+          </p>
+        </div>
+        <div class="conference-logo">
+          <img
+            src="./assets/Images/conference-logo.png"
+            alt="conference-logo"
+          />
+        </div>
+      </div>
+      <div class="main-bottom">
+        <div class="main-bottom-title">
+          <h3>Our Past Conference Keynote</h3>
+          <div class="underline"></div>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti
+            architecto quibusdam tempore voluptatibus dolor et aliquam amet
+            animi aspernatur quasi.
+          </p>
+        </div>
+        <div class="past-submit-container">
+          <div class="past-submit">
+            <div class="text-content">
+              <h2>2022</h2>
+              <p>
+                International Conference on Sustainable Water Management
+                (ICSWM-2022)
+              </p>
+              <div class="color-overlay"></div>
+            </div>
+          </div>
+          <div class="past-submit">
+            <div class="text-content">
+              <h2>2022</h2>
+              <p>
+                International Conference on Sustainable Water Management
+                (ICSWM-2022)
+              </p>
+              <div class="color-overlay"></div>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] On screen sizes with a width between 768px and 1200px this section is behaving like the mobile version with the two images aligned in a vertical column when they should be aligned in a horizontal row. Kindly change that to make it look like the template desktop version.
 - Your design:
 - 
<img width="1076" alt="image" src="https://user-images.githubusercontent.com/61919617/203633921-1315ecba-cd59-4bee-a689-d599cc95613b.png">